cmd/controller/main.go

```diff
@@ -157,6 +174,15 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Gateway")
 		os.Exit(1)
 	}
+
+	if err = (&dnshealthcheckprobe.DNSHealthCheckProbeReconciler{
+		Client:        mgr.GetClient(),
+		HealthMonitor: healthMonitor,
+		Queue:         healthCheckQueue,
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "DNSHealthCheckProbe")
+		os.Exit(1)
+	}
```

Comment:
What's the purpose of having this started here and in the health command? Is the intention to run that separately or as part of the controller, or is the health one to be run on cluster?

Reply:
The initial idea was to run it separately as a different container running in the spoke clusters, but we dropped that in favour of having it in the hub